evap/results/views.py

```diff
@@ -139,7 +139,7 @@ def add_warnings(course, course_result):
     # calculate the median values of how many people answered a questionnaire across all contributions
     questionnaire_max_answers = defaultdict(list)
     for questionnaire_result in course_result.questionnaire_results:
-        max_answers = max((question_result.total_count or 0 for question_result in questionnaire_result.question_results if question_result.question.is_rating_question), default=0)
+        max_answers = max((question_result.total_count for question_result in questionnaire_result.question_results if question_result.question.is_rating_question), default=0)
```

Comment:
removing the `or 0` leads to errors (e.g. results/semester/21/course/1577 in the test data)

Reply:
this shouldn't be the case, as total_count is only None if the rating results cannot be published, and that has been ruled out at the beginning of this method. have you refreshed the results cache? can you give me more info?